src/ui/components/Popover.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {Icon, useTheme, Caption} from '@ui/library';
+import {Popable} from 'react-native-popable';
+import {View, ViewStyle} from 'react-native';
+
+export type PopableProps = {
+  content: string;
+  contentStyle?: ViewStyle;
+  backgroundColor?: string;
+  icon?: string;
+  iconColor?: string;
+  iconSize?: number;
+  iconStyle?: ViewStyle;
```

Comment:
The component should have it's own basic style and be able to merge any incoming styles through props for specific needs.